pkg/detector/library/driver.go

```diff
@@ -135,19 +135,31 @@ func (d *Driver) DetectVulnerabilities(pkgID, pkgName, pkgVer string) ([]types.D
 }
 
 func createFixedVersions(advisory dbTypes.Advisory) string {
-	if len(advisory.PatchedVersions) != 0 {
-		return strings.Join(advisory.PatchedVersions, ", ")
-	}
-
 	var fixedVersions []string
-	for _, version := range advisory.VulnerableVersions {
-		for _, s := range strings.Split(version, ",") {
-			s = strings.TrimSpace(s)
-			if !strings.HasPrefix(s, "<=") && strings.HasPrefix(s, "<") {
-				s = strings.TrimPrefix(s, "<")
-				fixedVersions = append(fixedVersions, strings.TrimSpace(s))
+
+	uniqueVersions := make(map[string]int)
```

Comment:
I think we can just use [lo.Uniq(fixedVersions)](https://pkg.go.dev/github.com/samber/lo#readme-uniq)